### ISSUE TITLE
Pull command creates json files with an additional space on each line

### ISSUE THREAD

**Issue by baptisteArno** [tolgee/tolgee-cli#62]
Currently, when executing a pull command, it generates JSON files that looks like this:

```json
{
  "account.apiTokens.createButton.label" : "Erstellen"
}
```

IMO, It would be preferable to remove the space between the key and the column symbol:

```json
{
  "account.apiTokens.createButton.label": "Erstellen"
}
```

**Comment by cyyynthia** [comment]
The CLI doesn't process the files and dumps whatever it received from the Tolgee server; I believe that's something outside of the CLI scope here 😅

**Comment by baptisteArno** [comment]
Ok I see, thank you for the clarification

**Comment by allenhehe** [comment]
> The CLI doesn't process the files and dumps whatever it received from the Tolgee server; I believe that's something outside of the CLI scope here 😅

So this mean the json file generated by Tolgee server is wrong? It shouldn't contain those spaces right?

**Comment by allenhehe** [comment]
> Hello. It's already fixed by this PR: [tolgee/tolgee-platform#2432](https://github.com/tolgee/tolgee-platform/pull/2432)
> 
> However, it is valid json generated by the common jackskon objectMapper it wasn't a big issue before

I see, thank you!
